dumper/app/src/main/resources/oracle-stats/cdb/native/db-objects.sql

```diff
@@ -18,7 +18,8 @@ SELECT
   A.object_name "ObjectName",
   A.object_type "ObjectType",
   A.editionable "Editionable",
-  count(1) "Count"
+  count(1) "Count",
+  NULL "TableOwner"
```

Comment:
I think we shouldn't introduce TableOwner column to db-objects because it does not make sense in this context and won't be used by assessment. TableOwner is requires to filter synonyms only that are ingested to the separate temporary table. After the filtering assessment should take care about merging synonyms to db-objects.

Reply:
The column supplies no data, but on the Assessment end we rely on these two CSVs having the same structure.

Even after the filtering, this column will be present in the tmp table, so we'll need to specify which columns are getting merged and which aren't. I think allowing the redundant column with nulls is a safer option.

Reply:
I think either way this will be a non-trivial decision with some future consequences, so we should discuss it with the team before merging the PR or the CL.

Reply:
Let's not deviate from the dumper design principles that the tool should be as simple and as reliable as possible. It's better to add extra processing in the service than increase the complexity here.

Reply:
Colleagues, I see that we agreed on the statements:
1. Keep dumper as simple as possible
2. Add additional logic (complexity) on server if needed

Could you please clarify what specific actions follow from this?

Reply:
The discussion also took place in the ingestion code. The outcome is we don't add a null-filled column "TableOwner" in the dumper.

Reply:
The discussion here is a little out of date, the PR is currently on hold because we can't make a release at the moment.

I'll update it before reopening, but it's as Dominik said, the TableOwner will only be present in the synonym-public version of DB Objects.